README.md

```diff
@@ -40,21 +40,26 @@ $ age-plugin-tpm --generate -o age-identity.txt
 $ age-plugin-tpm -y age-identity.txt > age-recipient.txt
 
 # Encrypt / Decrypt something
-$ echo "Hack The Planet" | age -R ./age-recipient.txt -o test-decrypt.txt
-$ age --decrypt -i ./age-identity.txt -o - test-decrypt.txt
+$ echo 'Hack The Planet!' | age -R age-recipient.txt -o test-decrypt.txt
```

Comment:
Missing `!`, and single-quotes preferred for non-expanded strings

Reply:
I dropped `./`, as we don't use that for the other paths

Reply:
> Missing !

Lol, yes.